### PR TITLE
feat: updating home_directory to the correct location

### PR DESCRIPTION
**Pull request by nickelskevin**
**- What I did**
updated atctivate_cli.dart to use  "$homeDirectory/.atsign/keys";
**- How I did it**

**- How to verify it**

**- Description for the changelog**

feat: updating home_direcrtory to  "$homeDirectory/.atsign/keys";

### CODE REVIEW

File: packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart [hunk 1]
```diff
@@ -41,7 +43,10 @@ Future<void> main(List<String> arguments) async {
 
   stdout.writeln('[Information] Root server is ${argResults['rootServer']}');
 
-  var downloadPath = '${Directory.current.path}/keys';
+  // var downloadPath = '${Directory.current.path}/keys';
+  var homeDirectory = getHomeDirectory();
+  var downloadPath = "$homeDirectory/.atsign/keys";
```
Comment (gkc) [inline packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart:48]: Do you need to use platform-specific path separators? e.g. \ for Windows etc

**Review by gkc — COMMENTED** [review]
Are you intending a release? If so, please bump version in pubspec, and update CHANGELOG

File: packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart (hunk 1 above)
Comment (cconstab) [reply packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart:48]: We use the same code in attalk on windows. I really need to put those into a package..

**Review by cconstab — DISMISSED** [review]
Thank you

File: packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart (hunk 1 above)
Comment (cconstab) [reply packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart:48]: Updated ready for v1.2.5 release to pub.dev

**Review by cconstab — APPROVED** [review]
checked and tested manually

**Pull request merged**